src/frontend/screens/ObservationFields/QuestionLabel.tsx

```diff
@@ -1,21 +1,24 @@
 import * as React from 'react';
 import {View, StyleSheet} from 'react-native';
 import {FormattedFieldProp} from '../../sharedComponents/FormattedData';
-import {Text} from '../../sharedComponents/Text';
 import {Field} from '@comapeo/schema';
+import {HeaderText} from '../../sharedComponents/Text/HeaderText';
+import {Header} from 'react-native/Libraries/NewAppScreen';
 
 interface Props {
   field: Field;
 }
 
 export const QuestionLabel = ({field}: Props) => {
-  const hint = <FormattedFieldProp field={field} propName="placeholder" />;
+  // const hint = <FormattedFieldProp field={field} propName="placeholder" />;
```

Comment:
Does the commented code and console log have a purpose here? If not, can you please remove or adjust?

Reply:
https://github.com/digidem/comapeo-mobile/pull/849/commits/a5dd8f98482e4cddb5678275bdd2b98201ec362e